scipy/stats/_continuous_distns.py

```diff
@@ -7700,15 +7716,17 @@ def _entropy(self):
         for the root finder; the `scale` parameter and any other parameters
         for the optimizer are ignored.\n\n""")
     def fit(self, data, *args, **kwds):
+        if kwds.pop('superfit', False):
+            return super().fit(data, *args, **kwds)
```

Comment:
I've been adding this occasionally to `fit` methods because it allows us to activate the default `fit` method for comparison.